manifests/repository/apt.pp

```diff
@@ -5,15 +5,28 @@
 # === Parameters
 #
 class curator::repository::apt (
+  String $debian_major_version = $::os['release']['major'],
   String $version = $::curator::repository::version,
 ) inherits curator::repository {
 
   if ! defined( Class['::apt'] ) {
     class { '::apt': }
   }
 
+  $repo_suffix = $debian_major_version ? {
```

Comment:
I believe you can rely straight on the fact, I don't see any use case that would require updating it.

Reply:
The "new" repo is "debian9" while the previous one is "debian".  As far as I can see, "debian7", "debian8", etc are not valid, hence the extra variable.